### PR TITLE
[WIP] Dune 1.9 blog post

### PR DESCRIPTION
**Pull request by TheLortex**
Here is the blog post !
While testing locally I found some problems:
* it couldn't generate the blog index page
* lists look broken

Also is the picture mandatory ?

### CODE REVIEW

**Comment by emillon** [comment]
The picture is not mandatory but @diml loves a good picture :slightly_smiling_face:

**Comment by emillon** [comment]
The code sample at the end has wrong markup as well.

File: _posts/2019-04-10-dune-1-9-0.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## Coq support
+
+Dune now supports building Coq projects. To enable the experimental Coq
+extension, add `(using coq 0.1)` to your `dune-project` file. Then,
+you can use the `(coqlib ...)` stanza to declare Coq libraries.
+
+A typical `dune` file for a Coq project will look like:
+
+``
```
Comment (samoht) [inline _posts/2019-04-10-dune-1-9-0.md]: missing backtick

**Comment by TheLortex** [comment]
As this has been merged I've taken into account your reviews in https://github.com/ocaml/dune-www/pull/16

**Pull request merged**